pynear/tests/test_bktree.py

```diff
@@ -56,6 +56,7 @@ def test_bktree_find_self(bktree_cls, dimensions):
 
 
 @pytest.mark.parametrize("bktree_cls, dimensions", CLASSES)
+@pytest.mark.skip(reason="need fix")
```

Comment:
can you try to remove it again and see if it works now (since you merged in the bktree changes)

Reply:
Not working yet. In fact, some other test failed as well just one time. And then never failed again. :thinking: